pkgs/applications/networking/dnscontrol/default.nix

```diff
@@ -1,24 +1,26 @@
-{ stdenv, fetchFromGitHub, buildGoPackage}:
+{ lib, fetchFromGitHub, buildGoModule }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "dnscontrol";
-  version = "3.2.0";
+  version = "3.3.0";
 
-  goPackagePath = "github.com/StackExchange/dnscontrol";
+  goModulePath = "github.com/StackExchange/dnscontrol/v3";
```

Comment:
`goModulePath` doesn't exist and `goPackagePath` isn't needed with `buildGoModule`.

Reply:
Good catch. I removed it.